_data/api-swagger.yaml

```diff
@@ -11,18 +11,34 @@ schemes:
 produces:
   - application/json
 parameters:
-  AppkeyParam:
-    name: Application
-    description: The APPKey
+  AcceptHeader:
+    name: Accept
     in: header
     type: string
-    required: true
-  ApiKeyParam:
+    default: application/vnd.signhost.v1+json
```

Comment:
I  think default might be confusing, because:
1. it MUST be explicitly to use new features like the authentication array
2. this is only valid  for JSON

Reply:
> 2. this is only valid for JSON

Yes, that's why this parameter is only added to specific endpoints in the swagger file, so in that case default would make sense and it's also the way to add a value according to the spec. (Which I broke in the previous version of this PR 😅)

However I do agree that it might be confusing phrased in the documentation page itself.